ansible_builder/utils.py

```diff
@@ -20,8 +21,19 @@ def run_command(command, capture_output=False):
         sys.stdout.write(line)
 
     rc = process.poll()
-    if rc != 0:
-        print(MessageColors.FAIL + "An error occured, see output line(s) above for details." + MessageColors.ENDC)
+    if rc is not None and rc != 0:
+        print(MessageColors.FAIL + f"An error occured (rc={rc}), see output line(s) above for details." + MessageColors.ENDC)
```

Comment:
@beeankha for whatever reason, successful commands seem to sometimes give `None` for this, and because of that I was getting a lot of cases where perfectly good runs failed tests. I tried to look for some good documentation for why this was happening, and have come up empty handed.